### PR TITLE
Fix typos in overview

### PR DESCRIPTION
**Pull request by cnpryer**
Not necessarily typos, so if you'd like for me to update the commit I'd be happy to.

### CODE REVIEW

File: source/overview.rst [hunk 1]
```diff
@@ -300,7 +300,7 @@ Bringing your own Python executable
 
 Computing as we know it is defined by the ability to execute
 programs. Every operating system natively supports one or more formats
-of program they can natively execute.
+of the program they can natively execute.
```
Comment (chrysle) [inline source/overview.rst:303]:
```suggestion
of programs they can natively execute.
```

**Review by bhrutledge — APPROVED** [review]
Thanks!

**Comment by chrysle** [comment]
@cnpryer You should still make the change.

**Comment by cnpryer** [comment]
> @cnpryer You should still make the change.

I actually think I agree, but I mainly just wanted to point this out. If @bhrutledge agrees I'll push it (also have given maintainers push access).

**Comment by chrysle** [comment]
I think he has already agreed by enabling auto-merge (this won't be merged before all conversations are resolved).

**Pull request merged**